plugin/src/main/java/com/iridium/iridiumskyblock/upgrades/OresUpgrade.java

```diff
@@ -0,0 +1,16 @@
+package com.iridium.iridiumskyblock.upgrades;
+
+import com.cryptomorin.xseries.XMaterial;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+@NoArgsConstructor
+public class OresUpgrade extends UpgradeData {
+    public Map<XMaterial, Integer> ores;
```

Comment:
Maybe oreChances would be a better name?

Reply:
exactly

Reply:
meh it doesnt really matter?